packages/store/addon/-private/system/store.ts

```diff
@@ -54,6 +56,8 @@ import {
   CollectionResourceDocument,
   JsonApiDocument,
 } from '../ts-interfaces/ember-data-json-api';
+import { requestPromise } from './request-cache';
```

Comment:
symbols should be either CamelCase or CAPS_CASE given their absolute const nature